### ISSUE TITLE
Token values now adapt based on the device screen type

### ISSUE THREAD

**Issue by Tayebsed93** [Orange-OpenSource/ouds-flutter#137]
### Context:

To optimize the user interface (UI) based on phone orientation, size and grid tokens need to be dynamically adjusted. This ensures a consistent and smooth experience whether in portrait or landscape mode.

Size and grid tokens control spacing, margins, font sizes, and element layout. These values must be recalculated based on orientation to maximize space usage and maintain an aesthetic presentation.

### Detailed Description:

When the phone orientation changes, size and grid token values must be recalculated accordingly.

For example, grid column or row sizes and spacing may be wider in landscape mode to better use horizontal space, while portrait mode focuses more on vertical stacking of elements.

### Example : 
iOS Size token : https://github.com/Orange-OpenSource/ouds-ios/blob/develop/OUDS/Core/Themes/Orange/Sources/Values/SemanticTokens/OrangeTheme%2BSizeMultipleSemanticTokens.swift
iOS Space token : https://github.com/Orange-OpenSource/ouds-ios/blob/develop/OUDS/Core/Themes/Orange/Sources/Values/SemanticTokens/OrangeTheme%2BSpaceMultipleSemanticTokens.swift

**Comment by B3nz01d** [comment]
@Tayebsed93  I cannot test this issue until the size tokens are added in the token visualisation: #40

**Comment by B3nz01d** [comment]
Finished in Sprint 19
